.infrastructure/98_variables.tf

```diff
@@ -24,4 +24,80 @@ variable "tags" {
 ### required for security group ALB, ECS and RDS
 variable "cms_app_port" {
   default = 1337
+}
+
+variable "db_cluster_identifier" {
+  description = "Database Cluster identifier"
+  default     = "cms-database-cluster"
+}
+
+variable "db_name" {
+  description = "Database Name"
+  default     = "strapidb"
+}
+
+variable "db_client" {
+  description = "Database client"
+  default     = "postgres"
+}
```

Comment:
This variable is useless

Reply:
variable "db_client" it is used by the ECS task definition, which is present in the next PR. If it's not a problem I would leave it defined here

Reply:
with next commit remove this variable (and other)